pkgs/development/tools/misc/patchelf/unstable.nix

```diff
@@ -20,7 +20,7 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ autoreconfHook ];
   buildInputs = [ ];
 
-  doCheck = !stdenv.isDarwin;
+  doCheck = stdenv.hostPlatform.isMusl;
```

Comment:
@edolstra do you know why tests might be failing unless it is building for musl?

Reply:
@grahamc @edolstra how do we proceed from here? In my experience these tests didn't work for regular platforms even before this change (though not sure why; does NixOS CI periodically rebuild packages to validate it still works, incl. tests?)

Reply:
The tests should succeed, see for example https://hydra.nixos.org/build/94118663.

Reply:
We should merge this before 19.09, it seems like hydra doesn't build with musl though. Maybe that never worked?

Reply:
@domenkozar sorry for delay.

I've rebased the series on top of master. I don't know anything about Hydra. What I can say is that:

1) before the bump, default.nix had tests disabled;
2) before the bump, unstable.nix had tests enabled for !darwin;
3) after bump, I still can't pass the tests for default.nix but that's not regression;
4) it's unclear to me how to build the unstable.nix version instead of default.nix but I assumed that the exact same package should behave the same for default and stable (perhaps that's wrong assumption).

I wanted to check that tests still fail with unstable before the series and tried:

```
[ihar@dell:~/git/nixpkgs]$ nix-build . -A patchelf-0.10-pre-20190328
error: attribute 'patchelf-0' in selection path 'patchelf-0.10-pre-20190328' not found
```

Ideas how to build it?

Reply:
`nix-build -A patchelfUnstable` should work